docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.md

```diff
@@ -78,6 +79,42 @@ In compose mode you must call the [`saveAsync`](Office.context.mailbox.item.md#s
 ---
 ---
 
+#### masterCategories :[MasterCategories](/javascript/api/outlook/office.mastercategories)
```

Comment:
Nit for the future: This (and the others) should match the code format style of `name: Type` instead of `name :Type`.

Reply:
I'll add a task to scrub this throughout the Outlook OM docs

Reply:
Created OPX/Content task 3300044